### DISCUSSION TITLE
Functions didn't match well

### DISCUSSION THREAD

**Discussion by BillionaireBB97** [EddieHubCommunity/support#8305] [Community Call]
I don't know what's wrong.

**Comment by whisdev** [comment]
I will solve this problem.

**Reply by BillionaireBB97** [reply]
Thanks for your behaviour

**Comment by Vijaykv5** [comment]
May you can add a little bit more details..

**Comment by ShotaYamada9995** [comment]
Please explain about your issue more details...
